Vizzuality/landgriffon
# [LANDGRIF-1124]: Add user table permissions check

This PR adds a check to allow only admins to read and write  'profile/users' table 

### Designs

https://www.figma.com/file/7bmF29CIXDJwCpuloW5Nt6/%F0%9F%94%B5-Landgriffon---Redesign_BLUE?node-id=1870%3A167401&t=NSOfhtzvUZ20NwpC-4

### Testing instructions

1. Login as an ADMIN. Go to profile. 

- You should be able to click the users tab and go to 'profile/users'
- You should be able to see the users table

2. Login as USER. Go to profile.

- You should NOT be able to click on the 'users' tab.


### Related task
[LANDGRIF-1124](https://vizzuality.atlassian.net/browse/LANDGRIF-1124)

## Checklist before merging



[LANDGRIF-1124]: https://vizzuality.atlassian.net/browse/LANDGRIF-1124?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
